webapp/channels/src/components/admin_console/__snapshots__/database_settings.test.tsx.snap

```diff
@@ -387,7 +387,7 @@ exports[`components/DatabaseSettings should match snapshot 1`] = `
             className="help-text"
           >
             <MemoizedFormattedMessage
-              defaultMessage="Shows the currently active backend used for search. Values can be none, database, elasticsearch, bleve etc."
+              defaultMessage="Shows the currently active backend used for search. Values can be none, database, elasticsearch etc."
```

Comment:
Should we drop `etc`? (Obviously not in the snap, just commenting where I first saw this.)

```suggested
              defaultMessage="Shows the currently active backend used for search. Values can be none, database, or elasticsearch"
```